src/org/zaproxy/zap/extension/httpsinfo/ExtensionHttpsInfo.java

```diff
@@ -15,45 +15,49 @@
  * See the License for the specific language governing permissions and 
  * limitations under the License. 
  */
-package org.zaproxy.zap.extension.httpsInfo;
+package org.zaproxy.zap.extension.httpsinfo;
```

Comment:
OK. @kingthorin the following seems to work, could you try adding it to your branch?
```XML
diff --git a/build/build.xml b/build/build.xml
index f52d2bb..4a0315c 100644
--- a/build/build.xml
+++ b/build/build.xml
@@ -158,12 +158,13 @@
 
 	<macrodef name="build-addon" description="build the specified addon">
 		<attribute name="name"/>
+		<attribute name="addonid" default="@{name}" />
 		<element name="extra-actions" implicit="true" optional="true" />
 		<sequential>
 			<local name="zapaddon.version" />
 			<xmlproperty file="${src}/org/zaproxy/zap/extension/@{name}/ZapAddOn.xml"/>
 			<local name="file" />
-			<property name="file" value="@{name}-${status}-${zapaddon.version}.zap" />
+			<property name="file" value="@{addonid}-${status}-${zapaddon.version}.zap" />
 
 			<generatejavahelpsearchindexes jhalljar="${build.lib.dir}/jhall.jar"
 				helpcontentsdirname="contents" helpsetfilename="helpset*.hs">
@@ -230,7 +231,7 @@
 			</tstamp>
 
 			<appendzapaddonfile from="${src}/org/zaproxy/zap/extension/@{name}/ZapAddOn.xml" to="${versions.file}"
-				addonid="@{name}" filename="${file}" status="${status}" size="${length}" hash="${hash}" date="${yyyymmdd}"
+				addonid="@{addonid}" filename="${file}" status="${status}" size="${length}" hash="${hash}" date="${yyyymmdd}"
 				url="${zap.download.url}/${file}" />
 
 		</sequential>
@@ -238,12 +239,13 @@
 
 	<macrodef name="build-addon-without-help-indexes" description="build the specified addon">
 				<attribute name="name"/>
+				<attribute name="addonid" default="@{name}" />
 				<element name="extra-actions" implicit="true" optional="true" />
 				<sequential>
 					<local name="zapaddon.version" />
 					<xmlproperty file="${src}/org/zaproxy/zap/extension/@{name}/ZapAddOn.xml"/>
 					<local name="file" />
-					<property name="file" value="@{name}-${status}-${zapaddon.version}.zap" />
+					<property name="file" value="@{addonid}-${status}-${zapaddon.version}.zap" />
 
 					<local name="addon.libs.zip" />
 					<property name="addon.libs.zip" value="${temp}/libs-@{name}.zip" />
@@ -294,7 +296,7 @@
 					</tstamp>
 
 					<appendzapaddonfile from="${src}/org/zaproxy/zap/extension/@{name}/ZapAddOn.xml" to="${versions.file}"
-						addonid="@{name}" filename="${file}" status="${status}" size="${length}" hash="${hash}" date="${yyyymmdd}"
+						addonid="@{addonid}" filename="${file}" status="${status}" size="${length}" hash="${hash}" date="${yyyymmdd}"
 						url="${zap.download.url}/${file}" />
 
 				</sequential>
@@ -402,23 +404,25 @@
 
 	<macrodef name="build-deploy-addon" description="build and deploy the specified addon">
 		<attribute name="name"/>
+		<attribute name="addonid" default="@{name}" />
 		<sequential>
 			<antcall target="clean" />
 			<antcall target="compile" />
 
-			<build-addon name="@{name}" />
-			<deploy-addon name="@{name}" />
+			<build-addon name="@{name}" addonid="@{addonid}" />
+			<deploy-addon name="@{addonid}" />
 		</sequential>
 	</macrodef>
 
 	<macrodef name="build-deploy-addon-without-help-indexes" description="build and deploy the specified addon">
 		<attribute name="name"/>
+		<attribute name="addonid" default="@{name}" />
 		<sequential>
 			<antcall target="clean" />
 			<antcall target="compile" />
 
-			<build-addon-without-help-indexes name="@{name}" />
-			<deploy-addon name="@{name}" />
+			<build-addon-without-help-indexes name="@{name}" addonid="@{addonid}" />
+			<deploy-addon name="@{addonid}" />
 		</sequential>
 	</macrodef>
 	
@@ -518,7 +522,7 @@
 	</target>
   
   	<target name="deploy-httpsinfo" description="deploy the httpsInfo extension">
-		<build-deploy-addon name="httpsinfo" />
+		<build-deploy-addon name="httpsinfo" addonid="httpsInfo" />
 	</target>
 
   	<target name="deploy-httpsinfo-without-help-indexes" description="deploy the httpsInfo extension">
```